tests/unittests/gradCheckTest.cpp

```diff
@@ -169,36 +169,33 @@ TEST_P(InterpreterGrad, gradientCheckFCConcatRELU) {
   performGradCheck(EE_, result, A, Exp, &inputs, &outputs, 0.0001, 0.001);
 }
 
-static void gradientCheckGroupConv(size_t numInputChan, size_t group,
+static void gradientCheckGroupConv(size_t depth, size_t group,
                                    ExecutionEngine &EE_) {
   size_t numDim = 10;
-  size_t numOutputElem = 10;
 
   auto &mod = EE_.getModule();
   Function *F = mod.createFunction("main");
   auto *A =
-      mod.createVariable(ElemKind::FloatTy, {1, numDim, numDim, numInputChan},
+      mod.createVariable(ElemKind::FloatTy, {1, numDim, numDim, depth},
                          "A", VisibilityKind::Public, false);
-  auto *Ex = mod.createVariable(ElemKind::FloatTy, {1, numOutputElem}, "exp",
+  auto *Ex = mod.createVariable(ElemKind::FloatTy,
+                                {1, numDim + 1, numDim + 1, depth}, "exp",
                                 VisibilityKind::Public, false);
 
-  Node *O = F->createConv("conv", A, 4, 5, 1, 2, group);
```

Comment:
should we also fix: createAvgPool and other places?

Reply:
Yes. It is good idea to iterate over all grads covered by the test, and check that if grad is not doing anything, then test is indeed failing.